tests/profiler/test_profiler.py

```diff
@@ -331,7 +331,7 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
         files = [file for file in files if file.endswith('.json')]
         assert len(files) == 2, files
         local_rank = trainer.local_rank
-        assert any(f'{local_rank}-training_step_and_backward' in f for f in files)
+        assert any(f'{local_rank}-optimizer_step_and_closure_' in f for f in files)
```

Comment:
Any idea why it is `optimizer_step_and_closure_` and not `optimizer_step_and_closure` ?

Reply:
Well, both are fine because CI has passed.